site/src/component/Review/Review.tsx

```diff
@@ -160,8 +117,7 @@ const Review: FC<ReviewProps> = (props) => {
               key={review._id}
               course={props.course}
               professor={props.professor}
-              colors={getU(review._id) as VoteColor}
-              colorUpdater={updateVoteColors}
+              userVote={review.userVote!}
```

Comment:
I don't think we should need to assert non-null here. All reviews received from backend now have a vote associated with them, right? If so, we can update the type in `types.ts` so that userVote is required.

Reply:
Additionally, why specify it as a prop? We already pas the review prop so we can just do `review.userVote` within the SubReview component.

Reply:
Done